src/sage/combinat/k_regular_sequence.py

```diff
@@ -1221,11 +1263,24 @@ def some_inverse_U_matrix(lines):
                 raise RuntimeError
 
         def guess_linear_dependence(t_L, r_L, lines):
+            r"""
+                Based on an invertible submatrix of ``A`` as described in the
+                algorithm section of the docstring, find a candidate for a
+                linear combination of the rows of ``A`` yielding the subsequence
+                with parameters ``t_L`` and ``r_L``, i.e.,
+                `m \mapsto f(k**t_L * m + r_L)`.
+            """
             iU, m_indices = some_inverse_U_matrix(lines)
             X_L = vector(f(k**t_L * m + r_L) for m in m_indices)
             return X_L * iU
 
         def verify_linear_dependence(t_L, r_L, linear_dependence, lines):
+            r"""
+                Determine whether the subsequence with parameters ``t_L`` and
+                ``r_L``, i.e., `m \mapsto f(k**t_L * m + r_L)`, is the linear
+                combination ``linear_dependence`` of the current vector valued
+                sequence.
```

Comment:
```suggestion
                ``r_L``, i.e., `m \mapsto f(k**t_L * m + r_L)`, is the
                ``linear_dependence`` of the current vector valued
                sequence.
```

Reply:
@dkrenn : I do not think that "linear dependence" is more accurate than "linear dependence".